apps/dotcms-ui/src/app/shared/models/container/dot-container.model.ts

```diff
@@ -17,6 +17,10 @@ export interface DotContainer {
     };
 }
 
+export interface DotContainerStructure {
+    contentTypeVar: string;
+}
```

Comment:
You sure we don't have this? Like here: https://github.com/dotCMS/core-web/blob/issue-20664-apply-suggestions-design/libs/dotcms/src/lib/models/DotCMSPage.model.ts#L39

Reply:
we do, as we have `DotCMSContainer ` in that same file, but is another context